data/json/itemgroups/shops_trades.json

```diff
@@ -41,6 +41,7 @@
       [ "oxy_torch", 5 ],
       { "item": "weldtank", "prob": 2, "count": [ 1, 2 ] },
       { "item": "tinyweldtank", "prob": 2, "count": [ 1, 3 ] },
+	  { "item": "oxygen_cylinder", "prob": 10, "count": [ 1, 2 ] },
```

Comment:
```suggestion
      { "item": "oxygen_cylinder", "prob": 10, "count": [ 1, 2 ] },
```

Reply:
I suggest you change your editor to use spaces instead of tabs.

Reply:
Not a bad idea.